output/README.md

```diff
@@ -0,0 +1 @@
+default TPS workflow output directory
```

Comment:
Can we remove this file?  Hopefully the workflow is robust to a missing `output` directory and can create it if needed.  I see at `__make_out_folder` that looks like it does that.